File: test/fizzbuzzTest.php
<?php

require_once 'app/fizzbuzz.php';

class FizzBuzzTest extends PHPUnit_Framework_TestCase
{
  
  function testIsFizzBuzz() { 
    for ($i=1; $i<=100; $i = $i + 1) {
      if($i%15 == 0)
        $this->assertEquals("FizzBuzz", fizzbuzz($i));
      elseif($i%5 == 0)
        $this->assertEquals("Buzz", fizzbuzz($i));
      elseif($i%3 == 0)
        $this->assertEquals("Fizz", fizzbuzz($i));
      else 
        $this->assertEquals($i, fizzbuzz($i));
    }
  }

}

File: test/fizzbuzztest.py
from nose.tools import assert_equal

from app.fizzbuzz import fizzbuzz

expected = []
for i in range(1, 100):
	if(i % 15 == 0):
   		expected.append("FizzBuzz")
  	elif(i % 5 == 0):
    		expected.append("Buzz")
  	elif(i % 3 == 0):
    		expected.append("Fizz")
  	else: 
    		expected.append(str(i))

def test_fizzbuzz():
	for i in range(1, 100):
		assert_equal(fizzbuzz(i), expected[i-1])